<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab2d</class>
 <widget class="QWidget" name="tab2d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>475</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>File preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="twod_previewedit">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Datasets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_datasetsedit">
       <property name="toolTip">
        <string>A space separated list of dataset names to import from the file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Range of &amp;X:</string>
       </property>
       <property name="buddy">
        <cstring>twod_xminedit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="HistoryCombo" name="twod_xminedit">
       <property name="toolTip">
        <string>Enter a number for the minimum coordinate of the X axis (default 0)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="HistoryCombo" name="twod_xmaxedit">
       <property name="toolTip">
        <string>Enter a value for a maximum coordinate of the X axis (default is number of columns in file)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Range of &amp;Y:</string>
       </property>
       <property name="buddy">
        <cstring>twod_yminedit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="HistoryCombo" name="twod_yminedit">
       <property name="toolTip">
        <string>Enter a number for the minimum coordinate of the Y axis (default 0)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="HistoryCombo" name="twod_ymaxedit">
       <property name="toolTip">
        <string>Enter a number for the maximum coordinate of the Y axis (default number of rows in file)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="twod_invertrowscheck">
     <property name="text">
      <string>Invert &amp;rows when reading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="twod_invertcolscheck">
     <property name="text">
      <string>Invert colu&amp;mns when reading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="twod_transposecheck">
     <property name="text">
      <string>Trans&amp;pose rows and columns</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Data are read as a 2D matrix from the file.  X is read from left to right, Y from bottom to top, by default. These parameters listed here can be altered by including xrange A B, yrange A B, invertrows, invertcols and transpose as lines in the data file. Multiple datasets can be included by separating with blank lines.
</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
